assets/sass/components/idea-hub/_googlesitekit-idea-hub-dashboard-cta.scss

```diff
@@ -0,0 +1,58 @@
+/**
+ * Idea Hub DashboardCTA styles
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-idea-hub-dashboardcta {
+		background-color: $c-white;
+		display: flex;
+	}
+
+	.googlesitekit-idea-hub-dashboardcta__content {
+		display: inline-block;
+		padding: $grid-gap-desktop;
+
+		h5 {
+			font-family: $f-secondary;
+			font-size: 22px;
+			font-weight: $fw-secondary-normal;
+			margin: 0;
+			text-align: left;
+		}
+	}
+
+	.googlesitekit-idea-hub-dashboardcta__icon {
+		display: inline-block;
+		padding: $grid-gap-desktop 0 $grid-gap-desktop $grid-gap-desktop;
+	}
+
+	.googlesitekit-idea-hub-dashboardcta__icon svg {
+		background-color: rgba($c-idea-hub-orange, 0.2);
+		background-size: 144px 144px;
+		border-radius: 50%;
+		height: 144px;
+		padding: 32px;
```

Comment:
Is there a variable we can use for this value? If not, why 32px specifically?

Reply:
I've done something that made sense in the moment but I'm definitely more than happy to change it. I set the height and width of the SVG to 144px (I have now done this on the `BulbIcon` as you suggested), and gave it a circular `border-radius` and `background-colour` to achieve the 144px x 144px circle. 
The 32px padding then brings the icon down to the correct height:

| 144px x 144px circle | 32px padding |
| --- | --- |
| ![Screenshot 2021-05-26 at 10 31 08](https://user-images.githubusercontent.com/2470911/119637860-ff2e2a80-be0d-11eb-9f45-1de01eca9703.png) | ![Screenshot 2021-05-26 at 10 29 47](https://user-images.githubusercontent.com/2470911/119637853-fe959400-be0d-11eb-9947-605294c76eba.png) 


I can definitely understand that this might be hard to read. I was just trying to avoid a meaningless `<div>` or similar to get the circle.

Let me know what you think.

Reply:
Just noticed it was actually 30px in the design, so I'll update it and add a comment explaining the use of padding. Again, happy to change the approach here.

<img src=https://user-images.githubusercontent.com/2470911/119639331-69939a80-be0f-11eb-8faf-6939a98dfede.png width=200 />

Reply:
The style is specific to this component so it should be fine for now. If we need to use it in other places in different sizes we might want to use a percentage for the padding instead to preserve the proportions.